src/Configuration/PlatesFormatterConfiguration.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Equip\Configuration;
+
+use Auryn\Injector;
+use Equip\ContentNegotiation;
+use Equip\Formatter\PlatesFormatter;
+
+class PlatesFormatterConfiguration implements ConfigurationInterface
```

Comment:
Related: I'd be fine with moving this code to documentation so that it's not confused for code that's intended to be used as-is. In general, it should probably be a recommended practice to have a content negotiation configuration that's specific to the project in question.